braid/_braid.h

```diff
@@ -222,6 +222,8 @@ typedef struct
    braid_BaseVector  *va_alloc;      /**< original memory allocation for va */
    braid_BaseVector  *fa_alloc;      /**< original memory allocation for fa */
 
+   braid_BaseVector   ulast;         /**< stores vector at last time step */
```

Comment:
@rfalgout @steffi7574 @eric-c-cyr 
TorchBraid assumes that braid core always stores the last time point (whether F or C) in this variable ulast.  

Right now, ulast is only updated in Access and FCRelax when done==1, i.e., after the cycling. But this does move storage questions about vectors out of the Get/Set routines and into parts of the main code.  I'm not sure how I feel about this.  

We could we get rid of ulast inside of braid, and instead tell the user via status whether done==1, and whether this is the last time point.  Then, the user could always store ulast in app.  I'm not sure if this does everything Stefanie and Eric need in Torchbraid. 

Another alternative would be to just always store to ulast in the Get/Set routines, so that the extra logic for ulast is not needed in FCRelax or Access.  This does create a little more work during a Braid cycle, though.

Or, we could leave it as it is.

Reply:
Maybe we should just always store `ulast` and basically think of it as an extra C-point.  Then Get/Set only need to worry about it.  Do Get/Set have access to `done`?